controllers/comments.controller.js

```diff
@@ -34,3 +35,12 @@ exports.postCommentByArticleId = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+  removeCommentById(comment_id)
+    .then(() => res.status(204).send())
+    .catch((err) => {
+      next(err);
+    });
```

Comment:
A nifty little shorthand you can use for the catch is: `.catch(next);` and this will invoke `next` with the _error_ if it's gone to the catch 👍
For when there is nothing to send you can also use `.sendStatus(204);`